server/src/pages/shot/view.js

```diff
@@ -180,7 +180,7 @@ class Body extends React.Component {
 
   onClickDelete(e) {
     sendEvent("start-delete", "navbar", {useBeacon: true});
-    if (window.confirm("Are you sure you want to delete this shot permanently?")) {
+    if (window.confirm("Are you sure you want to delete this shot permanently?")) { // todo l10n: shotPageConfirmDeletion
```

Comment:
We are planning to use fluent-react, but for cases like this string, where a string is passed into `window.confirm` or `window.alert`, what's the right approach? Should we create an empty Localized tag in the DOM, then pass that into window.confirm?